.travis.yml

```diff
@@ -1,24 +1,29 @@
 sudo: false
 language: node_js
 node_js:
-  - 0.10
   - 0.12
-  - 4.4
-  - 5.8
+  - 4
+  - 5
+  - 6
 matrix:
-    allow_failures:
-        - node_js: 4.4
-        - node_js: 5.8
+  fast_finish: true
+  allow_failures:
+    - node_js: 6
 env:
   global:
     - CXX=g++-4.8
   matrix:
     - TEST_SUITE=units
     - TEST_SUITE=end-to-end
-script: "npm run $TEST_SUITE"
+script: npm run $TEST_SUITE
```

Comment:
Something we should do here is run the unit and end-to-end tests in the same travis run. Otherwise there is a ton of overhead for spinning up two travis jobs to do almost no extra work. Some of our repos have been calling `[npm run travis]`(https://github.com/pelias/openaddresses/blob/master/.travis.yml#L20) which is an NPM script that [runs both](https://github.com/pelias/openaddresses/blob/master/package.json#L39), and I think it works pretty well.